src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
It seems there is a syntax error due to a missing closing quotation mark in the console.log statement. Make sure to close the string with the appropriate quotation mark.